feature_engine/get_feature_names_out.py

```diff
@@ -5,9 +5,9 @@
 
 
 def _get_feature_names_out(
-        features_in: List[Union[str, int]],
```

Comment:
I think the only change in this file is the indentation, am I right? could we please remove it from the PR?